randovania/games/super_metroid/json_data/Ceres Station.txt

```diff
@@ -6,25 +6,31 @@ Extra - asset_id: 253
 Extra - save_index: 0
 > Door to Hallway A; Heals? False; Spawn Point
   * Normal Door to Hallway A/Door to Elevator Shaft
+  * Extra - different_strongly_connected_component: True
   > Escape
       After Fight Ridley At Ceres
 
 > Escape; Heals? False
   * Teleporter to Crateria - Landing Site
   * Extra - scan_asset_id: None
   * Extra - teleporter_instance_id: None
+  * Extra - different_strongly_connected_component: True
```

Comment:
You only need to add this flag into a single node of each component and its best to minimize the user to better catch errors.